.github/workflows/CI.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 env:
   CARGO_TERM_COLOR: always
-  MSRV: 1.56
+  MSRV: 1.60
```

Comment:
Eh does github trim trailing zeros... sigh

![image](https://user-images.githubusercontent.com/718941/183703492-84619cbc-6e94-419f-b6fa-75f3b1110a66.png)

Maybe this works

```suggestion
  MSRV: 1.60.0
```

Reply:
I think we just need to quote it so it's not interpreted as a number.